extensions/blocks/podcast-player/templates/playlist-track.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Podcast Title template.
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\Podcast_Player;
+
+// phpcs:disable VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+
+$track_title    = $attachment['title'];
+$track_duration = ! empty( $attachment['duration'] ) ? $attachment['duration'] : '';
+
+$class = 'jetpack-podcast-player__track ' . $secondary_colors['class'];
+$style = $secondary_colors['style'];
+if ( $is_active ) {
+	$class = 'jetpack-podcast-player__track is-active ' . $primary_colors['class'];
+	$style = $primary_colors['style'];
+}
+
+?>
+
+<li
+	class="<?php echo esc_attr( $class ); ?>"
+	style="<?php echo esc_attr( $style ); ?>"
+>
+	<a
+		class="jetpack-podcast-player__track-link"
+		href="<?php echo esc_url( $attachment['link'] ); ?>"
+		role="button"
+		aria-pressed="false"
```

Comment:
The `aria-pressed` line is getting removed by https://github.com/Automattic/jetpack/pull/15207 and replaced with `aria-current="track"` for the selected item and the attribute is removed for the rest of the items.

Reply:
Handled [here](https://github.com/Automattic/jetpack/pull/15230/commits/1f7fbb0217555a9d9c7c5770a592beea39fc25b7).